nixos/modules/programs/light.nix

```diff
@@ -17,11 +18,59 @@ in
           and udev rules granting access to members of the "video" group.
         '';
       };
+
+      brightnessKeys = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Whether to enable brightness control with keyboard keys.
+
+            This is mainly useful for minimalistic (desktop) environments. You
+            may want to leave this disabled if you run a feature-rich desktop
+            environment such as KDE, GNOME or Xfce as those handle the
+            brightness keys themselves. However, enabling brightness control
+            with this setting makes the control independent of X, so the keys
+            work in non-graphical ttys, so you might want to consider using this
+            instead of the default offered by the desktop environment.
+
+          '';
+        };
+
+        step = mkOption {
+          type = types.int;
+          default = 10;
+          description = ''
+            The percentage value by which to increase/decrease brightness.
+          '';
+        };
+
+      };
+
     };
   };
 
   config = mkIf cfg.enable {
     environment.systemPackages = [ pkgs.light ];
     services.udev.packages = [ pkgs.light ];
+    services.actkbd = mkIf config.programs.light.brightnessKeys.enable {
+      enable = true;
+      bindings = let
+        light = "${pkgs.light}/bin/light";
+        step = toString config.programs.light.brightnessKeys.step;
+      in [
+        {
+          keys = [ 224 ];
```

Comment:
These should probably be configurable.

Reply:
`sound.mediaKeys` doesn't offer options for that either, and I haven't found any issue or PR complaining about it. I guess the large majority of users just stick to the default keys (here, KEY_BRIGHTNESSDOWN and KEY_BRIGHTNESSUP), so I guess it would be of very little benefit to make them configurable.

I would rather see this PR finally merged, and maybe one day implement that in a separate one.